js/lib/authHelper.js

```diff
@@ -11,10 +11,15 @@ export const signOut = () => {
 export const isValidEmailAndPassword = (email, password) => {
   const validEmail = /^([\w-.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/;
 
-  const errorMessage = {
-    code: 'Invalid login',
-    message: 'Please Enter a Valid Email Address'
-  };
+  return validEmail.test(email) && password.length > 5;
+};
+
+export const errorMessage = {
```

Comment:
Move error messages into their own file named `errorMessages.js`.
Your error message file should export an object that looks like this: 
```js
export default {
    INVALID_LOGIN: {
        code: 'Invalid login',
        message: 'Please Enter a Valid Email Address'
    },
    USER_EXISTS: {
        code: 'User Already Exists',
        message: 'You Already Have an Account!'
    }
}
```

The keys of the object are the error constants you'll import where needed.